RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## New in git-machete 3.3.0
+
+- added: `hub` command with `anno-prs`, `create-pr` and `retarget-pr` subcommands
```

Comment:
Also, `Command properties/classification` in CONTRIBUTING.md needs to be updated